e2e/baseInvocationImage_test.go

```diff
@@ -0,0 +1,37 @@
+package e2e
```

Comment:
Not sure how I feel about the camel case naming of this file. I feel like the norm is more for using underscores, but I'm really not sure.

Reply:
Good point. I could'nt find any reference doc about file naming convention but it seems that snake case is much more popular than camel case. I renamed the file.